bin/verify-changelog-updated.js

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+// Load our dependencies
+var assert = require('assert');
+var fs = require('fs');
+
+// Load in our CHANGELOG
+var content = fs.readFileSync('CHANGELOG.md');
+
+// Verify we have a tag to find
+var version = process.env.FOUNDRY_VERSION;
+assert(version, 'Expected `env.FOUNDRY_VERSION` to be defined but it was not');
+
+// Verify our tag is present in the CHANGELOG
+if (content.indexOf(version) === -1) {
```

Comment:
I can see this being glitched if a version of another module (lets say a dependency) is bumped and we say

"4.2.0 - Bumped `yourlibraryhere` to 5.0.0"

When we get to `5.0.0` it would false positive?

Perhaps change to 
```js
content.indexOf(version + ' - ') === -1
```

To minimize the risk or even a regexp checking the version is at the start of a line